Epsilon.Host.Frontend/src/components/Competence/CompetenceProfile.vue

```diff
@@ -0,0 +1,77 @@
+<script setup lang="ts">
```

Comment:
As per Vue's conventions, template should come first, then script, then styling

Reply:
Oke, but the there needs to be a lint check. Beacuse Webstrom creates it in this order

Reply:
Rules added 
https://eslint.vuejs.org/rules/component-tags-order.html

Reply:
👍🏼